mqtt-client/main.py

```diff
@@ -422,7 +422,7 @@ def _load_sensor_configuration(self):
                 else:
                     ids = []
                     for config in result['config']:
-                        sensor_id = config['id']
+                        sensor_id = int(config['external_id'])
```

Comment:
external_id 's are not necessarily ints any longer. Plugins can also register external sensors based on e.g. unique strings.

See: https://github.com/openmotics/gateway/blob/48fe7ac0dee0bf2e0a31bc29a26fd86e46c29b58/src/gateway/models.py#L271

Reply:
Also see: https://github.com/openmotics/plugins/pull/49

The sensor ids are now split up and have 1 id per physical quanitity.

Reply:
@DaGolle you might want to check the mqtt consumer, the id's will have changed due to this split up. Also note that mqtt plugin version 2.0 was release to production today.